### PR TITLE
feat: Epg feeds

### PR DESCRIPTION
**Pull request by f-roland**
This PR adds randomized EPG feeds to build the EPG data

it has a few endpoints:

`/epg/channels(?count=x)` : returns x channels (default is 10, max available is 50)
`epg/programs?channel=<channelId>(&start_time=<date>&timezone=<timezone>)`: returns programs for a channel
Gives programs for 24 hours from the given date (now if not given), and adjusted for the requested timezone (utc by default)

Programs are randomized but put in a cache so that subsequent requests for the same channelId on the same starting time will return consistent data.

cache can be viewed by going to `/epg/cache` and cleared by going to `/epg/clear_cache`

### CODE REVIEW

File: src/epg/channels.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
+const path = require("path");
+const fs = require("fs");
+const R = require("ramda");
+const md5 = require("md5");
+
+const capitalize = R.converge(R.concat, [R.compose(R.toUpper, R.head), R.tail]);
```
Comment (Budaa) [inline src/epg/channels.js:6]: Creative one ;D
`R.replace(/^./, R.toUpper)` will work as well.

**Review by Budaa — APPROVED** [review]
LGTM 👍🏻

File: src/epg/channels.js (hunk 1 above)
Comment (Budaa) [reply src/epg/channels.js:6]: Yep
https://ramdajs.com/repl/?v=0.28.0#?R.replace%28%2F%5E.%2F%2C%20R.toUpper%2C%20%27foo%20foo%20foo%27%29%3B%20%2F%2F%3D%3E%20%27bar%20foo%20foo%27%0A%0A

**Review by Budaa — APPROVED** [review]

**Review by gavriguy — APPROVED** [review]
@f-roland not sure that the cache is really needed but as you already implemented it I think its fine.

**Comment by f-roland** [comment]
@gavriguy we'll see. What I wanted to avoid is the situation where the data changes every time a request is made (i.e. the screen reloads)

this will address this and I believe will help with development

**Pull request merged**